homeassistant/components/rainbird/switch.py

```diff
@@ -2,60 +2,100 @@
 
 import logging
 
+from pyrainbird import AvailableStations, RainbirdController
 import voluptuous as vol
 
 from homeassistant.components.switch import PLATFORM_SCHEMA, SwitchDevice
 from homeassistant.const import (
+    ATTR_ENTITY_ID,
     CONF_FRIENDLY_NAME,
     CONF_SCAN_INTERVAL,
-    CONF_SWITCHES,
     CONF_TRIGGER_TIME,
     CONF_ZONE,
 )
 from homeassistant.helpers import config_validation as cv
 
-from . import DATA_RAINBIRD
+from . import DATA_RAINBIRD, DOMAIN, RAINBIRD_CONTROLLER
 
-DOMAIN = "rainbird"
 _LOGGER = logging.getLogger(__name__)
 
+ATTR_DURATION = "duration"
+
+SERVICE_START_IRRIGATION = "start_irrigation"
+
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
Remove the platform schemas. They will not be used when loading the integration via the component.

Reply:
platform schema removed.